Gemfile

```
@@ -17,7 +17,7 @@ gem "bgs", git: "https://github.com/department-of-veterans-affairs/ruby-bgs.git"
 gem "bootsnap", require: false
 gem "browser"
 gem "business_time", "~> 0.9.3"
-gem "caseflow", git: "https://github.com/department-of-veterans-affairs/caseflow-commons", ref: "6377b46c2639248574673adc6a708d2568c6958c"
+gem "caseflow", git: "https://github.com/department-of-veterans-affairs/caseflow-commons", branch: "AlecK/APPEALS-27013"
```

🎗️ **Reminder:** After `caseflow-commons 1.0.0` is merged, update this line and `bundle update caseflow`:

```suggestion
gem "caseflow", "~> 1.0.0", git: "https://github.com/department-of-veterans-affairs/caseflow-commons"
```